sycl/test-e2e/ProgramManager/multi_device_bundle/compile_link.cpp

```diff
@@ -1,7 +1,7 @@
 // REQUIRES: gpu && linux && (opencl || level_zero)
 
 // RUN: %{build} -o %t.out
-// RUN: env NEOReadDebugKeys=1 CreateMultipleRootDevices=3 SYCL_UR_TRACE=2 %{run} %t.out
```

Comment:
`lit` captures stdout/stderr, so why waste our machine resources for something that is not used anyways?